Evolutionary_loop/Loop_Parts/Part_B/Part_B_VPol_job2.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+#*******************************************************************************
+#  Original file name: Part_B_GPU_job2_asym_array.sh
+#	 		This is Part B2 of the loop, which prepares and runs output.xmacro with 
+#			the relevant parameter (antenna type, population number, grid size, etc)
+#			Output.xmacro writes the XFDtd simulation data to an output file (.uan)
+#
+#  Programmer: OSU GENETIS Team
+#
+#  Revision history:
+#     07/13/23  Jason Yao, third pass over the loop for training purposes
+#
+#  Notes:
+# 		* vertical ruler at column 80; tab size 2
+#
+#
+#  TODO:
+#
+#*******************************************************************************
+
+# varaibles
```

Comment:
```suggestion
# variables
```

This file seems to duplicate most of the code of the previous one, so the same comments apply.  If possible it would be better to avoid duplication altogether.

Reply:
done.
as of now, three previous VPol files have been merged into the current script, and one is moved in the archived.
I update the archived file to keep it up to date, but eventually I'll check with the group to see if we want to get rid of it.

I'll also check with the group to see how we can merge PUEO and VPol scripts into one script to avoid duplication.